### PR TITLE
[v4 DEV] A working chart in ChartJs 3.9.1

### PR DESCRIPTION
**Pull request by devinmatte**
Screen Shot 2022-10-18 at 12 49 22 PM

### CODE REVIEW

**Comment by devinmatte** [comment]
This is Demo/Dev work, no need to fully review. There will be a full review at a later date

File: pages/SingleDayLineChart.tsx [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    if (point[benchmark_field] === null) {
+      return '#1c1c1c'; //grey
+    } else if (ratio <= 1.25) {
+      return '#64b96a'; //green
+    } else if (ratio <= 1.5) {
+      return '#f5ed00'; //yellow
+    } else if (ratio <= 2.0) {
+      return '#c33149'; //red
+    } else if (ratio > 2.0) {
+      return '#bb5cc1'; //purple
```
Comment (skaplan-dev) [inline pages/SingleDayLineChart.tsx:55]: could we have these as constants somewhere?

**Review by skaplan-dev — APPROVED** [review]

**Comment by austinjpaul** [comment]
No more fill for the benchmark?

File: pages/SingleDayLineChart.tsx (hunk 1 above)
Comment (devinmatte) [reply pages/SingleDayLineChart.tsx]: Yeah we could. I just copy pasted from the current `src`. I'll move these to a better place

**Comment by devinmatte** [comment]
It doesn't seem to be taking @austinjpaul. Still trying to figure out what the new attributes needed to make that work again

**Pull request merged**